vortexasdk/operations.py

```diff
@@ -79,7 +79,5 @@ def search(self, **params) -> List[dict]:
         >>> Search("/reference/vessels").search(term="DHT") # doctest: +SKIP
 
         """
-        logger.info(
```

Comment:
Previously we logged the payload before elements were filtered. This was  not helpful - as we weren't actually logging the payload sent to the API